web-components/src/components/timepicker/TimePicker.ts

```diff
@@ -60,42 +62,33 @@ export namespace TimePicker {
 
 @customElement("md-timepicker")
 export class TimePicker extends LitElement {
+  @property({ type: Boolean }) twoDigitAutoTab = false;
```

Comment:
all booleans basically need to also have an attribute friendly name (no camelCase)